spec/numberformat.html

```diff
@@ -457,10 +457,11 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have a [[boundFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
+        1. If _nf_.[[initializedNumberFormat]] is *undefined*, then
```

Comment:
last time I checked `[[initializedNumberFormat]]`, it was a flag to indicate that the instance was correctly created (soft of a dummy flag). As for this condition here, it is just to define the boundFormat internal slot once (the first time .format is used).